src/ui-presets/playback.js

```diff
@@ -65,6 +66,7 @@ export function playbackUI(props: any): React$Element<any> {
       {PlaylistCountdown.shouldRender(props) ? <PlaylistCountdown player={props.player} /> : undefined}
       <PrePlaybackPlayOverlay player={props.player} />
       <CastBeforePlay player={props.player} />
+      <PictureInPictureOverlay player={props.player} />
```

Comment:
validate overlapping. e.g. large play button, playlist end screen

Reply:
that ok with me - thanks!